### PR TITLE
Yank node exporter

### PR DESCRIPTION
**Pull request by dbrinegar**
fixes lightbend/es-backend#527

### CODE REVIEW

File: enterprise-suite/es-grafana/cluster-exporter.json [hunk 1]
```diff
@@ -1,7 +1,44 @@
...
+         "format": "bytes"
+      }
+   },
+   {
+      "graphName": "Bytes Received Per Second",
+      "promQL": [
+         "sum by (namespace) (rate(container_network_receive_bytes_total[1m]))"
+      ],
+      "yaxes": {
+         "format": "bytes"
```
Comment (marcoderama) [inline enterprise-suite/es-grafana/cluster-exporter.json]: Should that be "bytes/s"?

File: enterprise-suite/es-grafana/cluster-exporter.json [hunk 2]
```diff
@@ -1,7 +1,44 @@
...
+      }
+   },
+   {
+      "graphName": "Bytes Transmitted Per Second",
+      "promQL": [
+         "sum by (namespace) (rate(container_network_transmit_bytes_total[1m]))"
+      ],
+      "yaxes": {
+         "format": "bytes"
```
Comment (marcoderama) [inline enterprise-suite/es-grafana/cluster-exporter.json]: Ditto.

File: enterprise-suite/es-grafana/cluster-exporter.json [hunk 3]
```diff
@@ -1,7 +1,44 @@
 [
-   {"graphName": "CPU Percent", "promQL":["node_cpu_percent"]},
-   {"graphName": "Memory Available Percent", "promQL":["node_memory_usable_percent"]},
-   {"graphName": "File System Free Percent", "promQL":["node_filesystem_free_percent"]},
-   {"graphName": "Bytes Received Per Second", "promQL":["sum by (instance) (irate(node_network_receive_bytes[1m]))"]},
-   {"graphName": "Bytes Transmitted Per Second", "promQL":["sum by (instance) (irate(node_network_transmit_bytes[1m]))"]}
+   {
+      "graphName": "CPUs Used",
```
Comment (marcoderama) [inline enterprise-suite/es-grafana/cluster-exporter.json]: As discussed with @dbrinegar offline, I found this name confusing.  I naively thought it meant "number of CPUs dedicated to the container" and _could not_ figure out how that could be derived from `system_seconds`.  I see now that it's a measure of CPU utilization and that it's the standard k8s nomenclature...

File: enterprise-suite/es-grafana/cluster-exporter.json (hunk 1 above)
Comment (dbrinegar) [reply enterprise-suite/es-grafana/cluster-exporter.json]: the y-axis here is just converting to human readable MB GB kinds of terms. the Per Second is in title

**Pull request merged**